lib/Models/CatalogGroup.js

```diff
@@ -82,7 +82,22 @@ var CatalogGroup = function(terria) {
         }
     };
 
-    knockout.track(this, ['isOpen', 'isLoading', 'items']);
+    knockout.track(this, ['_isOpen', 'isLoading', 'items']);
+
+    knockout.defineProperty(this, 'isOpen', {
```

Comment:
What not just subscribe to change notification on `isOpen` instead of overriding the property?  In fact, we're already subscribing to it below, so hopefully you can just add some more logic to the existing handler.